vscode/src/context/openctx/remoteRepositorySearch.ts

```diff
@@ -13,24 +13,27 @@ const RemoteRepositorySearch: Provider & {
 
     async mentions({ query }) {
         try {
-            const dataOrError = await graphqlClient.searchRepos(10, undefined, query)
+            const dataOrError = await graphqlClient.searchRepos(30, undefined, query)
 
             if (isError(dataOrError) || dataOrError === null) {
                 return []
             }
 
             const repositories = dataOrError.repositories.nodes
 
-            return repositories.map(
-                repo =>
-                    ({
-                        uri: repo.url,
-                        title: repo.name,
-                        data: {
-                            repoId: repo.id,
-                        },
-                    }) as Mention
-            )
+            return repositories
+                .map(repo =>
+                    contextFiltersProvider.isRepoNameIgnored(repo.name)
+                        ? null
+                        : {
+                              uri: repo.url,
+                              title: repo.name,
+                              data: {
+                                  repoId: repo.id,
+                              },
+                          }
+                )
+                .filter(isDefined)
```

Comment:
would this read nicer if we did:

```suggestion
            return repositories
                .filter(repo => contextFiltersProvider.isRepoNameIgnored(repo.name))
                .map(repo =>
                    ({
                              uri: repo.url,
                              title: repo.name,
                              data: {
                                  repoId: repo.id,
                              },
                          })
                )
```

Reply:
not filtering out now